backends/dpdk/dpdkArch.h

```diff
@@ -987,6 +993,40 @@ class ElimHeaderCopy : public Transform {
     const IR::Node* postorder(IR::Member* m) override;
 };
 
+class EmitDpdkTableConfig : public Inspector {
```

Comment:
We should  add a comment that this pass reuses the code from P4RuntimeEntriesConverter

Reply:
done